content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -128,7 +128,7 @@ different Kubernetes components.
 | `ServerSideApply` | `false` | Alpha | 1.14 | 1.15 |
 | `ServerSideApply` | `true` | Beta | 1.16 | |
 | `ServiceNodeExclusion` | `false` | Alpha | 1.8 | |
-| `StartupProbe` | `true` | Beta | 1.17 | |
+| `StartupProbe` | `false` | Alpha | 1.16 | |
```

Comment:
Hello @Sh4d1 .
Looks like StartupProbe is an _alpha_ feature according to:

https://deploy-preview-19079--kubernetes-io-master-staging.netlify.com/docs/reference/generated/kubernetes-api/v1.17/#container-v1-core

https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#when-should-you-use-a-startup-probe

/lgtm